ansible/openshift-playbook.yml

```diff
@@ -14,9 +15,11 @@
 
   - name: Openshift pre-flight checks
     include_tasks: tasks/pre-flight-checks-openshift.yml
+    when: not test_harness_dry_run
```

Comment:
How about using a block with when instead of repeating it all the time? Then it would be less likely that someone forgets to add this to new tasks in the future.